drainer/syncer.go

```diff
@@ -61,6 +66,11 @@ type Syncer struct {
 
 	shutdown chan struct{}
 	closed   chan struct{}
+
+	// TODO: to discuss block allow list implementation in the future
+	// baList       *baf.Filter
```

Comment:
Does this mean we want to use baList to replace the  do-dbs/tables ignore-dbs/tables?

Reply:
I will delete this part. As we discussed before, we will **NOT** use baList to replace the do-dbs/tables ignore-dbs/tables.